sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveUdfs.scala

```diff
@@ -321,6 +323,18 @@ private[hive] case class HiveGenericUdtf(
   override def toString = s"$nodeName#${funcWrapper.functionClassName}(${children.mkString(",")})"
 }
 
+/**
+ * Resolve Udtfs Alias.
+ */
+private[spark] object ResolveUdtfsAlias extends Rule[LogicalPlan] {
+  def apply(plan: LogicalPlan) = plan transform {
+    case q: LogicalPlan => q transformExpressions {
+      case Alias(udtf@HiveGenericUdtf(_, _, _), names) if names.contains("|") =>
+        Alias(udtf.copy(aliasNames = names.split("\\|")), names)()
```

Comment:
I believe this will work, but seems a little hacky to me. A better way probably is create a new expression node like `MultiAlias`. And since only single UDTF expression is supported the SELECT clause, we'd better also add a semantic checking in `analyzer`.
What do you think?